control/control_standalone_test.go

```diff
@@ -1,166 +0,0 @@
-// +build linux,standalone
-
-package control
-
-import (
-	"bytes"
-	"io"
-	"io/ioutil"
-	"os"
-	"path/filepath"
-	"testing"
-
-	"github.com/containerd/containerd/namespaces"
-	"github.com/moby/buildkit/cache"
-	"github.com/moby/buildkit/cache/metadata"
-	"github.com/moby/buildkit/client"
-	"github.com/moby/buildkit/snapshot"
-	"github.com/moby/buildkit/snapshot/blobmapping"
-	"github.com/moby/buildkit/source"
-	"github.com/moby/buildkit/source/containerimage"
-	"github.com/moby/buildkit/worker"
-	"github.com/moby/buildkit/worker/runcworker"
-	"github.com/stretchr/testify/assert"
-	"golang.org/x/net/context"
-)
-
-func TestControl(t *testing.T) {
```

Comment:
Where is this test now? Would be good to keep something that makes sure that the different components work together.